### ISSUE TITLE
Multiple instances don't work

### ISSUE THREAD

**Issue by dlewand691** [RickStrahl/jquery-resizable#22]
I'm building a style guide and would like to use this for many instances across the page. Currently when dragging one, all the others resize and when releasing the mouse, the first stays active. How can this be fixed? 

Here is a pen using the demo and showing the issue: https://codepen.io/danlewski/pen/rwzrvP

**Comment by RickStrahl** [comment]
If you want different behavior from those two separate panels you'll have to name them differently. Use a unique ID for left and right panels and the splitter. 

The issue here is there's no way to discern directly which item the sizing applies to, because the library doesn't know the relationship between the splitter that's firing the events and the target other than the selector you give it.

**Comment by dlewand691** [comment]
Ah, ok. That makes sense. The only problem is that I'm going to have 50+ instances and there's no way to maintain that. Could targeting the one being clicked on with `this` possibly work? So resize `this` on mouseDown, reset on mouseUp?

**Comment by ArnoutCreemers** [comment]
@dlewand691, I have the same requirement to maintain multiple split panes on one page. 
I fixed it by finding the correct handle for each resizeable pane and binding the events only to that specific handle. 

I made some changes to the getHandle function to get this to work:
```
function getHandle(selector, $el) {
        if (selector && selector.trim()[0] === ">") {
            selector = selector.trim().replace(/^>\s*/, "");
            return $el.find(selector);
        }

        return selector ? ($el).parent().find(selector) : $el;
    } 
```

**Comment by RickStrahl** [comment]
Thanks Arnout, this seems to work great to filter the set down! As long as there's a non-common container this will work.
I've merged this change in. Thank you.

**Comment by RickStrahl** [comment]
Released in 0.26.
